tsconfig.json

```
@@ -8,7 +8,7 @@
       "outDir": "./dist",
       "rootDir": "./",
       "sourceMap": true,
-      "target": "ESNext",
+      "target": "ES6",
       "moduleResolution": "node"
```

```suggestion
      "moduleResolution": "node",
      "strict": true
```
Recommended, a TS szerint is https://www.typescriptlang.org/tsconfig#strict